testing/docker.go

```diff
@@ -18,17 +18,22 @@ import (
 	dockerclient "github.com/docker/docker/client"
 )
 
-func NewDockerContainer(t testing.TB, image string, env []string) (*DockerContainer, error) {
+func NewDockerContainer(t testing.TB, image string, env []string, cmd []string) (*DockerContainer, error) {
 	c, err := dockerclient.NewEnvClient()
 	if err != nil {
 		return nil, err
 	}
 
+	if cmd == nil {
+		cmd = make([]string, 0)
+	}
+
 	contr := &DockerContainer{
 		t:         t,
 		client:    c,
 		ImageName: image,
 		ENV:       env,
+		Cmd: 	   cmd,
```

Comment:
Cockroach's docker image doesn't auto-start the DB, since the image serves as the DB, SQL client, administrative utilities, etc.

In order to actually run the DB for testing, it needs to be executed with the command `start --insecure`.  As such, the Cmd functionality of the docker container API was added.